src/tools/clippy/clippy_lints/src/lib.rs

```diff
@@ -1965,7 +1965,6 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
         LintId::of(absurd_extreme_comparisons::ABSURD_EXTREME_COMPARISONS),
         LintId::of(approx_const::APPROX_CONSTANT),
         LintId::of(async_yields_async::ASYNC_YIELDS_ASYNC),
-        LintId::of(atomic_ordering::INVALID_ATOMIC_ORDERING),
```

Comment:
You should also remove the `atomic_ordering.rs` module from Clippy. All the other Clippy changes LGTM.

Reply:
We could do a manual sync

Reply:
Yeah it also wouldn't hurt to do it a few days earlier. But lets wait for the FCP first.

Reply:
> You should also remove the atomic_ordering.rs module from Clippy.

Thanks, done. I'd actually removed the module already, I just forgot to remove the file :laughing: